src/test/java/com/zendesk/maxwell/schema/columndef/ColumnDefTest.java

```diff
@@ -143,6 +144,40 @@ public void TestStringAsJSON() {
 	}
 
 	@Test
+	public void TestJSON() {
+		byte input[] = new byte[14];
+		input[0] = Byte.valueOf((byte) 0);
+		input[1] = Byte.valueOf((byte) 1);
+		input[2] = Byte.valueOf((byte) 0);
+		input[3] = Byte.valueOf((byte) 13);
+		input[4] = Byte.valueOf((byte) 0);
+		input[5] = Byte.valueOf((byte) 11);
+		input[6] = Byte.valueOf((byte) 0);
+		input[7] = Byte.valueOf((byte) 2);
+		input[8] = Byte.valueOf((byte) 0);
+		input[9] = Byte.valueOf((byte) 5);
+		input[10] = Byte.valueOf((byte) 3);
+		input[11] = Byte.valueOf((byte) 0);
+		input[12] = Byte.valueOf((byte) 105);
+		input[13] = Byte.valueOf((byte) 100);
```

Comment:
Seems a little verbose, can you use `new byte[] { ... }` instead of setting each individual byte?